beacon_node/src/lib.rs

```diff
@@ -67,6 +68,20 @@ impl<E: EthSpec> ProductionBeaconNode<E> {
         let blobs_db_path = client_config.create_blobs_db_path()?;
         let executor = context.executor.clone();
 
+        // Log panics properly.
+        {
+            let log = log.clone();
+            std::panic::set_hook(Box::new(move |info| {
+                crit!(
+                    log,
+                    "Task panic. This is a bug!";
```

Comment:
Should it suggest to contact developers / open a Github issue if this happens?

At least like in the previous version
```
                            "advice" => "Please check above for a backtrace and notify \
                                         the developers"
```

Reply:
yeah makes sense to keep it, thanks!